src/system_window.cpp

```diff
@@ -291,10 +297,11 @@ SystemWindow::SystemWindow( Settings& settings )
 			SDL_WINDOWPOS_CENTERED, SDL_WINDOWPOS_CENTERED,
 			width, height,
 			( (is_opengl || use_gl_context_for_software_renderer_) ? SDL_WINDOW_OPENGL : 0 ) | ( fullscreen ? SDL_WINDOW_FULLSCREEN : 0 ) | SDL_WINDOW_SHOWN );
-
 	if( window_ == nullptr )
 		Log::FatalError( "Can not create window" );
 
+	SDL_SetWindowKeyboardGrab(window_, SDL_TRUE);
```

Comment:
```
error: use of undeclared identifier 'SDL_SetWindowKeyboardGrab'
        SDL_SetWindowKeyboardGrab(window_, SDL_TRUE);
        ^
```
This doesn't exist in SDL 2.0.9.

Reply:
The controls don't work without KeyboardGrab which is available since SDL 2.0.16.